tests/app/shared/issue-tables/issue-sorter.spec.ts

```diff
@@ -49,15 +54,18 @@ describe('issuer-sorter', () => {
       assertOrder(sortedIssuedByIdDesc, dummyIssue, otherDummyIssue);
     });
 
-    it('sorts issues based on their todos left correctly', () => {
-      matSort.active = 'Todo Remaining';
+    it('sorts issues based on their updated date fields correctly', () => {
+      const issueUpdatedEarlier: Issue = Issue.createPhaseBugReportingIssue(ISSUE_UPDATED_EARLIER);
+      const issueUpdatedLater: Issue = Issue.createPhaseBugReportingIssue(ISSUE_UPDATED_LATER);
+      matSort.active = 'date';
       matSort.direction = 'asc';
-      const sortedIssuesByTodoAsc = getSortedData(matSort, todoIssuesList);
-      assertOrder(sortedIssuesByTodoAsc, otherModerationIssue, moderationIssue);
+
+      const sortedIssuedByIdAsc = getSortedData(matSort, [issueUpdatedEarlier, issueUpdatedLater]);
```

Comment:
I think this constant is mistakenly named? Also, you can put this issue list as a constant at the start of the describe block, similar to the now deleted lines 14-16 which i've screenshotted below:
<img width="941" alt="Screenshot 2024-02-07 at 8 24 45 PM" src="https://github.com/CATcher-org/WATcher/assets/68138671/cecfea38-8c4d-4178-82a9-37d9258a4e2f">